cwl_utils/sandboxjs.py

```diff
@@ -308,6 +308,8 @@ def new_js_proc(
         if nodejs is None or nodejs is not None and required_node_version is False:
             try:
                 nodeimg = "docker.io/node:slim"
+                singularityimgs = [0]
+                dockerimgs = '\n'
```

Comment:
This makes `dockerimgs == "\n"` a "magic value" , by using it to indicate the engine is singularity, which is a bit hard to understand. I would remove this, and adapt the later if statement instead (see comment there).